apps/condo/domains/notification/constants/constants.js

```diff
@@ -342,7 +342,8 @@ const MESSAGE_META = {
     },
     [CUSTOM_CONTENT_MESSAGE_TYPE]: {
         dv: { required: true },
-        title: { required: true },
+        title: { required: false },
```

Comment:
It looks strange. You should have one parameter.

Reply:
added ticket to Jira DOMA-4130